tools/Config.mk

```diff
@@ -241,10 +241,12 @@ ifeq ($(CONFIG_WINDOWS_NATIVE),y)
   INCDIR ?= "$(TOPDIR)\tools\incdir.bat"
 else ifeq ($(CONFIG_CYGWIN_WINTOOL),y)
   DEFINE ?= "$(TOPDIR)/tools/define.sh" -w
-  INCDIR ?= "$(TOPDIR)/tools/incdir$(HOSTEXEEXT)" -w
+  INCDIR ?= $(shell if [ -f $(TOPDIR)/tools/incdir${HOSTEXEEXT} ];\
+              then echo "$(TOPDIR)/tools/incdir$(HOSTEXEEXT)" -w; else echo ":";fi)
```

Comment:
wildcard(https://www.gnu.org/software/make/manual/html_node/Wildcard-Function.html) is more simple and portable.